Re-review of the four docs (#1756)
spryker/spryker-docs: docs/scos/dev/guidelines/keeping-a-project-upgradable/supported-extension-scenarios/event-subscribers-registration.md

    link: docs/scos/dev/guidelines/keeping-a-project-upgradable/supported-extension-scenarios/modules-configuration.html
---

Manifests support registering event subscribers only in the dependency provider, a type of code class.

> Manifests fully support the registration of event subscribers in the collection. Restrictions to the order of the plugins in collection are **not supported**. New plugin will be added to the end of the collection.

Reviewer: use italic to dray attention to smth: https://developers.google.com/style/text-formatting

> Manifests fully support the registration of event subscribers in the collection. Restrictions to the order of the plugins in collection are **not supported**. New plugin will be added to the end of the collection.

Author: ```suggestion
Manifests fully support the registration of event subscribers in the collection. Restrictions to the order of the plugins in collection are *not supported*. New plugin will be added to the end of the collection.
```

    }
}
```

> Manifests fully support the registration of event subscribers in the collection. Restrictions to the order of the plugins in collection are *not supported*. New plugin will be added to the end of the collection.

Reviewer: ```suggestion
Manifests fully support the registration of event subscribers in the collection. Restrictions to the order of the plugins in collection are *not supported*. New plugin is added to the end of the collection.
```
